pyproject.toml

```diff
@@ -71,13 +71,15 @@ dependencies = [
   "black>=23.7.0",
   "mypy>=1.0.0",
   "ruff>=0.0.243",
+  "typos"
 ]
 [tool.hatch.envs.lint.scripts]
 typing = "mypy --install-types --non-interactive {args:src/cycax tests}"
 style = [
   "ruff {args:.}",
   "black --check --diff {args:.}",
 ]
+spell = "typos {args:.}"
```

Comment:
The {args:.} defaults to the current directory of given path. This is useful if you want to scan only some file or wish to provide extra flags to typo.

e.g.`hatch run lint:spell src/cycax/cycad/engines/`